src/main/webapp/restaurant-page.html

```diff
@@ -0,0 +1,73 @@
+<!--
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Restaurant Page</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/styles.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <script src="https://cdn.ckeditor.com/ckeditor5/11.2.0/classic/ckeditor.js"></script>
+    <script src="/js/restaurant-page-loader.js"></script>
+    <style>
+      /* Form is hidden by default. */
+      .hidden {
+        display: none;
+      }
+    </style>
+  </head>
+  <body onload="buildUI();">
+    <nav>
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/aboutus.html">About Our Team</a></li>
+        <li><a href="/stats.html">Statistics</a></li>
+      </ul>
+    </nav>
+    <h1 id="page-title">User Page</h1>
+
+    <div id="about-me-container">Loading...</div>
+    <div id="about-me-form" class="hidden">
+      <form action="/about" method="POST">
+        <textarea name="about-me" id="about-me-input" placeholder="about me"></textarea>
+        <br/>
+        <input type="submit" value="Submit">
+      </form>
+    </div>
+
+    <div class="message-form-container">
+    <div id="message-form" class="hidden">
```

Comment:
The blobstore message posting action requires a specific action/name without a div in order to post a message. If you replace the second div with: `<form id="message-form" class="hidden" method="POST" enctype="multipart/form-data">` I think it will allow messages to post without throwing an exception.